### ISSUE TITLE
Extensions stopped working

### ISSUE THREAD

**Issue by Profesor08** [microsoft/vscode#217906]
Type: <b>Bug</b>

After last update, extensions stopped working. There the list os extensions:

- stylelint
- prettier
- source control (!)
- and many others

VS Code version: Code - Insiders 1.91.0-insider (3e9cab05385c88a3174175e47085952bd3caf8a3, 2024-06-25T05:54:05.606Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 9 3900XT 12-Core Processor            (24 x 3793)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|31.92GB (16.04GB free)|
|Process Argv|--crash-reporter-id 15de8306-e8a1-4213-8c02-ebcaa1dda863|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (28)</summary>

Extension|Author (truncated)|Version
---|---|---
glance|ahk|0.0.8
project-manager|ale|12.8.0
lit-html|bie|1.11.1
vscode-markdownlint|Dav|0.55.0
vscode-eslint|dba|3.0.10
intelli-php-vscode|DEV|0.12.15062
phptools-vscode|DEV|1.48.15635
profiler-php-vscode|DEV|1.48.15635
gitlens|eam|15.1.0
EditorConfig|Edi|0.16.4
prettier-vscode|esb|10.4.0
auto-close-tag|for|0.5.15
auto-rename-tag|for|0.1.10
todo-tree|Gru|0.0.226
path-autocomplete|ion|1.25.0
auto-comment-blocks|kev|1.0.1
dotenv|mik|1.0.1
remote-wsl|ms-|0.88.2
hexeditor|ms-|1.10.0
vscode-extension-auto-import|Nuc|1.4.3
indent-rainbow|ode|8.3.1
vscode-xml|red|0.27.2024053103
vscode-yaml|red|1.15.0
vscode-styled-components|sty|1.7.8
vscode-stylelint|sty|1.4.0
vscode-mdx|uni|1.8.9
highlight-matching-tag|vin|0.11.0
vscode-import-cost|wix|3.3.0

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256197
vsc_aa:30263845
vscod805cf:30301675
vsaa593cf:30376535
py29gd2263:31024238
c4g48928:30535728
2i9eh265:30646982
962ge761:30841072
pythongtdpath:30726887
welcomedialog:30812478
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
dsvsc015:30821418
pythonregdiag2:30926734
pythonmypyd1:30859725
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
cppperfnew:30980852
pythonait:30973460
jchc7451:31067543
chatpanelt:31014475
945dj816:31013170
a69g1124:31018687
dvdeprecation:31040973
dwnewjupytercf:31046870
newcmakeconfigv2:31071590
nb_pri_only:31057983
nativerepl2:31071685

```

</details>

### Stylelint

```
[Error - 12:07:29 PM] Stop failed after configuration change
  Message: Pending response rejected since connection got disposed
  Code: -32097 
[Error - 12:07:29 PM] Server process exited with code 1.
````

### Prettier

```
["INFO" - 12:01:57 PM] Extension Name: esbenp.prettier-vscode.
["INFO" - 12:01:57 PM] Extension Version: 10.4.0.
["INFO" - 12:01:57 PM] Extension is disabled. No formatters will be registered. To enable, change the prettier.enable to true and restart VS Code.
```

### Source Control

![image](https://github.com/microsoft/vscode/assets/8218016/0b6b3b83-b90b-46c6-a56b-667e315e2768)

**Comment by wiesys** [comment]
I had the same issue, but luckily it looks like it's just fixed (at least it works again for me) in this version:

Version: 1.91.0-insider (Universal)
Commit: b36286db80c68b7fdc6bfee2f55e0208c8833dfc
Date: 2024-06-25T09:17:43.016Z

**Comment by gjsjohnmurray** [comment]
The Start Extension Bisect command can help diagnose this sort of issue.

**Comment by Profesor08** [comment]
May be it is fized with latest update

**Comment by wiesys** [comment]
Yes, for me the issue is fixed with commit b36286db80c68b7fdc6bfee2f55e0208c8833dfc :)
